modules/solace/solace.go

```diff
@@ -0,0 +1,125 @@
+package solace
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/go-connections/nat"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	defaultVpn = "default"
+)
+
+type SolaceContainer struct {
```

Comment:
suggestion: let's use a simpler name. I know there are other modules using the "old" name (including the module name), but we are slowly moving to a place where the package/module adds the context: `solace.Container` reads better than `solace.SolaceContainer`.

```suggestion
type Container struct {
```

Reply:
done